packages/base/src/config/Icons.js

```diff
@@ -0,0 +1,60 @@
+import { getTheme, isThemeFamily } from "./Theme.js";
+
+const IconCollectionConfiguration = new Map();
+
+const registerConfig = (theme, collectionName) => {
+	IconCollectionConfiguration.set(theme, collectionName);
```

Comment:
IMO we can merge this function with the one using it:
```js
const setDefaultIconCollection = (theme, collectionName) => {
	if (collectionName === "horizon") {
		collectionName = "SAP-icons-v5";
	}
	IconCollectionConfiguration.set(theme, collectionName);
};
```

Reply:
done